tools/docker-dev/Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM docker:latest as docker
+FROM docker:stable as docker
 
-FROM ubuntu:latest
+FROM ubuntu:xenial
 ARG goversion=1.12.5
 ARG user
 ARG group
 ARG uid=1000
 ARG gid=1000
 
 # Install development packages.
-RUN apt-get update && apt-get -qqy upgrade && apt-get -qqy install \
+# hadolint ignore=DL3008
```

Comment:
Not needed anymore if you removed the upgrade?

Reply:
The DL3008 is about the required pinned version.
https://github.com/hadolint/hadolint/wiki/DL3008